File: program/convert-miles-to-inches/ConvertMilesToInches.java
import java.util.Scanner;

public class ConvertMilesToInches {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int miles;
        int inches;

        System.out.print("Input: ");
        miles = input.nextInt();

        input.close();

        inches = miles * 63360;

        System.out.printf("Output: %d", inches);
        System.out.println();
    }
}

File: program/program/convert-miles-to-inches/ConvertMilesToInches.kt
import java.util.Scanner

fun main() {
    print("Enter String : ")
    println()
    var min = readLine()

    println("$min miles into inches = " + ConvertMilesToInches(min!!.toDouble()))

}

private fun ConvertMilesToInches(miles: Double): Double {
    var inches = miles * 63360

    return inches
}
